test/test_ugrid.py

```diff
@@ -12,7 +12,7 @@
     [
         helpers.tri_mesh,
         helpers.quad_mesh,
-        helpers.quad_tri_mesh,
+        helpers.tri_quad_mesh,
```

Comment:
Let's test both tri_quad and quad_tri here.

Reply:
ok no problem

Reply:
it actually helped to find some issues and improve the code. Thanks for the idea.